services/director/Dockerfile

```diff
@@ -47,16 +47,21 @@ ENV SC_BUILD_TARGET build
 
 # Installing client libraries and any other package you need
 #
-# - client library for PostgreSQL https://www.postgresql.org/docs/9.5/libpq.html
+#  libpq: client library for PostgreSQL https://www.postgresql.org/docs/9.5/libpq.html
+#  libstdc++: needed in ujson https://github.com/kohlschutter/junixsocket/issues/33
 #
 RUN apk update && \
     apk add --no-cache \
-      libpq
+      libpq \
+      libstdc++
+
+
+
 
 # Installing build dependencies (will be deleted in production)
 RUN apk add --virtual .build-deps \
       git \
-      gcc \
+      g++ \
```

Comment:
Are you sure that removing this using apk-del .build-deps acutally reduces the image size?

Reply:
No it does not. I will remove that

Reply:
All good. Keep in mind you talk to somebody whose base image is 2.0 GB large...